samples/nrf9160/secure_services/src/main.c

```diff
@@ -63,6 +63,10 @@ void main(void)
 	}
 
 	print_hex_number(buf, num_bytes_to_read);
+#elif defined(PM_S0_ADDRESS)
+	bool valid = spm_prevalidate_b1_upgrade(PM_S0_ADDRESS, PM_S0_ADDRESS);
```

Comment:
Why is the same address given twice?

Reply:
Because the function takes two addresses, current and target, but here we call the function on the current b1 for demonstration purposes.